src/dataframe/dataframe.jl

```diff
@@ -596,10 +605,33 @@ Base.setindex!(df::DataFrame, x::Void, col_ind::Int) = delete!(df, col_ind)
 
 Base.empty!(df::DataFrame) = (empty!(df.columns); empty!(index(df)); df)
 
-function Base.insert!(df::DataFrame, col_ind::Int, item::AbstractVector, name::Symbol)
+# TODO: add docstring of makeunique after 0.4 release of DataFrames
```

Comment:
As noted elsewhere, the docstring should be added now.

Reply:
added